ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/mapper/claimantresponse/CourtDeterminationMapper.java

```diff
@@ -1,11 +1,12 @@
-package uk.gov.hmcts.cmc.ccd.deprecated.mapper.claimantresponse;
+package uk.gov.hmcts.cmc.ccd.domain.mapper.claimantresponse;
```

Comment:
please move this into top mapper package

Reply:
please check my last commit for this PR.